Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:18
+FROM node:24
```

Comment:
Only question I have is whether it would be less risky to upgrade this to the latest LTS while still running the matrix tests to make sure we're ready to go all the way to the latest/current/active ... but defer 100% to you on that.